python/src/lib.rs

```diff
@@ -26,13 +26,28 @@ impl DeltaTable {
         Ok(self._table.version)
     }
 
-    pub fn files(&self) -> PyResult<Vec<String>> {
-        Ok(self._table.get_files().to_vec())
+    #[args(full_path = "false")]
+    pub fn files(&self, full_path: bool) -> PyResult<Vec<String>> {
+        let list = self._table.get_files().to_vec();
+        if full_path {
+            Ok(list
+                .iter()
+                .map(|fpath| {
+                    Path::new(&self._table.table_path)
```

Comment:
I did a lot of this last weekend, getting `String` from `Path` is super annoying

Reply:
yes, utf8 fun.